descartes_utilities/src/spline_interpolator.cpp

```diff
@@ -0,0 +1,232 @@
+/**
```

Comment:
Is this the code that is BSD? If so, I think it needs to be labeled BSD similar to the others that are marked Apache.